ui/client/components/Fields.js

```diff
@@ -143,12 +150,18 @@ export default class Fields extends React.Component {
                  removeField={this.removeField.bind(this)}
                  moveItem={moveItem}
                  showValidation={this.props.showValidation}
```

Comment:
Myaby better will be extract all props data?

Reply:
Done